Source/Terminals.Plugins.Web/TerminalsWebExport.cs

```diff
@@ -0,0 +1,34 @@
+﻿namespace Terminals.Integration.Export
+{
+    internal class TerminalsHTTPExport : ITerminalsOptionsExport
+    {
+        public void ExportOptions(IExportOptionsContext context)
+        {
+            if (context.Favorite.Protocol.Contains("HTTP"))
+                TerminalsWebExport.ExportOptions(ref context);
+        }
+    }
+
+    internal class TerminalsHTTPSExport : ITerminalsOptionsExport
```

Comment:
No need to have to exporters, if they do the same. We may implement one for both protocols. See ConnectionManager.IsProtocolWebBased.

Reply:
Correcting this :)